frontend/packages/console-shared/src/components/dashboard/resource-quota-card/AppliedClusterResourceQuotaItem.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  getQuotaResourceTypes,
+  QuotaScopesInline,
+  QuotaGaugeCharts,
+} from '@console/internal/components/resource-quota';
+import { ResourceLink } from '@console/internal/components/utils/resource-link';
+import { AppliedClusterResourceQuotaModel } from '@console/internal/models';
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
+
+import './resource-quota-card.scss';
+
+const AppliedClusterResourceQuotaItem: React.FC<AppliedClusterResourceQuotaItemProps> = ({
+  resourceQuota,
+  namespace,
+}) => {
+  const resourceTypes = getQuotaResourceTypes(resourceQuota);
+  const scopes = _.get(resourceQuota, 'spec.scopes');
```

Comment:
Prefer optional chaining for new code.

```suggestion
  const scopes = resourceQuota?.spec?.scopes;
```